### PR TITLE
Version no added

### PR DESCRIPTION
**Pull request by sam-a-d**

### CODE REVIEW

**Comment by s-amad** [comment]
Can you please clear with some meta information?

**Comment by sam-a-d** [comment]
Ok. i'll get back to you.

**Comment by s-amad** [comment]
sure. see you

File: li.txt [hunk 1]
```diff
@@ -1,2 +1,5 @@
 // Comment added for this file.
```
Comment (s-amad) [inline li.txt:1]: remove // please.

**Review by s-amad — CHANGES_REQUESTED** [review]

File: li.txt (hunk 1 above)
Comment (sam-a-d) [reply li.txt:1]: Done

**Comment by sam-a-d** [comment]
removed // please check

**Pull request merged**